Added JWT service logic

MarianoInc opened the pull request:
**Summary**

- Added jwt IToken provider.
- Added jwt Token provider. 
- Added jwt authentication service.
![Interfaz jwt provider](https://user-images.githubusercontent.com/87680683/163653412-4c546202-b3a4-4004-b8a1-099bce0ebef7.PNG)
![Lógica de creación del token](https://user-images.githubusercontent.com/87680683/163653430-50c1f8ad-5dc3-4bdb-b311-af4c5e187974.PNG)
![Lógica del token jwt](https://user-images.githubusercontent.com/87680683/163653461-60cb9546-8572-4cb3-aa5d-6ff005c9c542.PNG)

MarianoInc committed:
Added JWT service logic

emmanuelranone reviewed (changes requested):
Lo que hiciste está bien. solamente faltaría agregar una clase con el método para obtener el token con sus respectivos claims. Esa clase podes crearla en el helper

MarianoInc committed:
Merge branch 'dev' into add_JWT_Configuration/UT188-38
Branch update

MarianoInc committed:
Added JwtTokenProvider class, IJwtTokenProvider interface.
Authentication IssuerSigningKey Refactor.
Added DevelopmentApiKey to appsettings.json.

MarianoInc committed:
Added CreateJwtToken function logic to JwtTokenProvider

MarianoInc commented:
Añadí la clase que lo genera con una interfaz

emmanuelranone committed:
Merge branch 'dev' into add_JWT_Configuration/UT188-38
